CHANGELOG.md

```diff
@@ -0,0 +1,78 @@
+# Changelog
```

Comment:
Opt to manually create changelog file content, then I'll check if we can auto-generate the changelog for versions as we're already using [conventional commit message](https://www.conventionalcommits.org/en/v1.0.0/).